### PR TITLE
Add a paddle version of DDPG

### PR DESCRIPTION
**Pull request by ShuaibinLi**
### Add files:
mujoco_agent.py, mujoco_model.py, train.py, README.md, ---> benchmark/paddle/DDPG/
ddpg.py ---> parl/algorithms/paddle/

### Change file:
parl/algorithms/paddle/init.py

### CODE REVIEW

File: benchmark/paddle/DDPG/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Benchmark result
+
+<img src=".benchmark/DDPG_results.png" width = "800" height ="400" alt="DDPG_results"/>
++ Each experiment was run three times with different seeds
+
+## How to use
+### Dependencies:
++ python3.5+
++ [parl==1.4.1](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline benchmark/paddle/DDPG/README.md]: parl >= 1.4.2

File: benchmark/paddle/DDPG/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
++ gym==0.9.1
++ [paddlepaddle==2.0.0](https://github.com/PaddlePaddle/Paddle)
```
Comment (rical730) [inline benchmark/paddle/DDPG/README.md]: paddlepaddle>=2.0.0

File: benchmark/paddle/DDPG/train.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```
Comment (rical730) [inline benchmark/paddle/DDPG/train.py]: 2020 --> 2021

File: benchmark/paddle/DDPG/train.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--env", default="HalfCheetah-v1", help='OpenAI gym environment name')
+    parser.add_argument(
+        "--seed",
+        default=0,
+        type=int,
+        help='Sets Gym, PyTorch and Numpy seeds')
```
Comment (rical730) [inline benchmark/paddle/DDPG/train.py]: remove Pytorch and Numpy

Comment (rical730) [reply benchmark/paddle/DDPG/train.py]: seeds --> seed

File: parl/algorithms/paddle/ddpg.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+__all__ = ['DDPG']
+
+
+class DDPG(parl.Algorithm):
+    def __init__(self,
+                 model,
+                 gamma=None,
+                 tau=None,
+                 actor_lr=None,
+                 critic_lr=None):
```
Comment (rical730) [inline parl/algorithms/paddle/ddpg.py:29]: Please add descriptions for parameters

File: benchmark/paddle/DDPG/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+### Mujoco games introduction
+Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco games.
+
+### Benchmark result
+
+<img src=".benchmark/DDPG_results.png" width = "800" height ="400" alt="DDPG_results"/>
++ Each experiment was run three times with different seeds
+
+## How to use
+### Dependencies:
```
Comment (rical730) [inline benchmark/paddle/DDPG/README.md:15]: Dependencies  recommended order:
python
parl
paddle
gym
mujoco

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
Looks good to me.

**Pull request merged**